### ISSUE TITLE
Karfreitag and Ostermontag have names in English

### ISSUE THREAD

**Issue by frexvahi** [daviskirk/german-holidays#3]
All the other holidays in the returned DataFrame have the German names, but Good Friday and Easter Monday have the English names.

**Comment by daviskirk** [comment]
I agree.  Since the package is called german holidays, we can probably go with all german names
